server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/method/MailboxSetMethod.scala

```diff
@@ -140,7 +140,7 @@ case class UpdateResults(results: Seq[UpdateResult]) {
   }).toMap
 }
 
-class MailboxSetMethod @Inject()(serializer: Serializer,
+class MailboxSetMethod @Inject()(serializer: MailboxSerializer,
```

Comment:
Same remark

Reply:
Idem, its not an object as there is a dependency to the MailboxId.Factory